.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
```

Comment:
Is this necessary? For Windows you would want `crlf`.

Reply:
You don't necessarily need `crlf` under windows. Most editors support `lf` as well. Git itself is build to use `lf` for newlines. It can be configured to convert `crlf` to `lf` on check in but I'm not sure why you'd want to do that when you can work directly with `lf`.

Reply:
`git` by default checks out files with crlf. If the editor starts using if in a file that is formatted with crlf then that will cause problems.

Reply:
The way `git` checks out files is dependent on the way was configured during installation. I guess adding a `.gitattributes` to make this uniform would be a good idea indeed.

Reply:
@IntelOrca 

> git by default checks out files with crlf.

This is 100% incorrect. Git by default checks out whatever is in repository and only because you've set your `core.autocrlf = true` it does that.

Reply:
@janisozaur I wasn't specific. If you install git for Windows and do not change any of the initial configuration then check out this repository, you will get CRLF line endings.

Reply:
Agree with letting Git handle this.

@manuelVo Could you take this line out?